cranelift/filetests/filetests/isa/s390x/call.clif

```diff
@@ -48,6 +48,7 @@ block0(v0: i32):
 ; block0:
 ;   lgr %r5, %r2
 ;   llgfr %r2, %r5
+;   lgr %r2, %r5
```

Comment:
Hi @elliottt  this seems a bug: the `lgr` overwrites the effect of the `llgfr`.   Is there some problem with the extension handling on return values?